### PR TITLE
Ethereals now have hair

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
add: Ethereals have started growing slightly transparent hair in a new mutation of the lifeform.
/:cl:

one of the main pieces of feedback I got was lack of customization, this will slightly remedy this but I'd still like to look for more unique options.
image
image
image

### CODE REVIEW

**Comment by wesoda25** [comment]
Gonna be perfectly honest I don't think this is the type of customization ethereals need

**Comment by Ziiro** [comment]
That looks really cool, actually. I like it.

**Comment by Tlaltecuhtli** [comment]
now they are ghetto slime girl

**Comment by Qustinnus** [comment]
https://streamable.com/1lmfl

**Comment by trollbreeder** [comment]
this PR is a tie in with #49932

**Comment by optimumtact** [comment]
make it so if they're emagged they change hairstyle too

**Comment by Qustinnus** [comment]
🤔 thats a funny idea

**Comment by AdamElTablawy** [comment]
it's an electrically powered wig.

**Comment by Qustinnus** [comment]
wrong pr you're looking for #49932

**Comment by ATH1909** [comment]
> make it so if they're emagged they change hairstyle too

Nah, I think just the color changing without the hairshape changing is fine. Sometimes, you just wanna play a rainbow boi with an massive afro instead of a rainbow boi with some sort of eldritch, shapeshifting, hair-like abomination on his head.

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -374,6 +374,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 				if(hair_color)
 					if(hair_color == "mutcolor")
 						facial_overlay.color = "#" + H.dna.features["mcolor"]
+					else if(hair_color == "fixedmutcolor")
```
Comment (bobbahbrown) [inline code/modules/mob/living/carbon/human/species.dm]: maybe make fixedmutcolor a defined constant

**Pull request merged**